Formula/sane-backends.rb

```diff
@@ -1,18 +1,24 @@
 class SaneBackends < Formula
   desc "Backends for scanner access"
   homepage "http://www.sane-project.org/"
-  url "https://fossies.org/linux/misc/sane-backends-1.0.24.tar.gz"
-  mirror "https://mirrors.kernel.org/debian/pool/main/s/sane-backends/sane-backends_1.0.24.orig.tar.gz"
-  sha256 "27c7085a54f1505d8b551e6f1e69d30e1ee57328b18429bb2225dabf4c45462d"
-  bottle do
-    revision 1
-    sha256 "e8cd147368ca911b15da016a09cb3d0b58843b5169291a75fe2a42fed7c9c887" => :el_capitan
-    sha256 "006f73ab657625408f5d56ec1596498b911781164db60cc4518370b2a08686f3" => :yosemite
-    sha256 "168d5dc5f38b6e568f2d757de77dc86c7b3bab1a84a50f1b30103eb9ba9bf367" => :mavericks
-    sha256 "483533ac9a7d48d15350afaed266ac9472cd8c945fdd34d610253253e3c384e7" => :mountain_lion
-  end
 
-  revision 1
+  head "https://anonscm.debian.org/cgit/sane/sane-backends.git"
+
+  stable do
+    url "https://fossies.org/linux/misc/sane-backends-1.0.25.tar.gz"
+    mirror "https://mirrors.kernel.org/debian/pool/main/s/sane-backends/sane-backends_1.0.25.orig.tar.gz"
+    sha256 "a4d7ba8d62b2dea702ce76be85699940992daf3f44823ddc128812da33dc6e2c"
+    bottle do
+      revision 1
+      sha256 "e8cd147368ca911b15da016a09cb3d0b58843b5169291a75fe2a42fed7c9c887" => :el_capitan
+      sha256 "006f73ab657625408f5d56ec1596498b911781164db60cc4518370b2a08686f3" => :yosemite
+      sha256 "168d5dc5f38b6e568f2d757de77dc86c7b3bab1a84a50f1b30103eb9ba9bf367" => :mavericks
+      sha256 "483533ac9a7d48d15350afaed266ac9472cd8c945fdd34d610253253e3c384e7" => :mountain_lion
+    end
+    # Fixes some missing headers missing error. Reported upstream
+    # https://lists.alioth.debian.org/pipermail/sane-devel/2015-October/033972.html
+    patch :DATA
```

Comment:
Please submit this to https://github.com/Homebrew/formula-patches & use that link instead. We're just asking for trouble carrying around a big blob of diff inside a formula.